articles/app-service/monitor-instances-health-check.md

```diff
@@ -20,9 +20,9 @@ Note that _/api/health_ is just an example added for illustration purposes. We d
 ## What App Service does with Health checks
 
 - When given a path on your app, Health check pings this path on all instances of your App Service app at 1-minute intervals.
-- If an instance doesn't respond with a status code between 200-299 (inclusive) after 10 requests, App Service determines it's unhealthy and removes it from the load balancer for this Web App. The required number of failed requests for an instance to be deemed unhealthy is configurable to a minimum of two requests.
+- If a Web App running on a given instance doesn't respond with a status code between 200-299 (inclusive) after 10 requests, App Service determines it's unhealthy and removes it from the load balancer for this Web App. The required number of failed requests for an instance to be deemed unhealthy is configurable to a minimum of two requests.
```

Comment:
Hi Cris!

Please use lower case here: web app

Reply:
Done. Thank you.